src/pages/CardList.js

```diff
@@ -0,0 +1,26 @@
+import Card from '../components/CardPlus';
+import CardPlus from '../components/CardPlus';
+import PageTitle from '../components/PageTitle';
+import RootContainer from '../components/container/Root';
+import AppContainer from '../components/container/App';
+import { useContext } from 'react';
+import { AppContext } from '../AppContext';
+import uuid from 'react-uuid';
+
+const CardList = () => {
+  const { cardList = [] } = useContext(AppContext);
+
+  return (
+    <RootContainer>
+      <AppContainer appClass="flex-column-center">
+        <PageTitle title="보유 카드" />
+        {cardList.map((card, i) => (
+          <Card key={uuid()} card={card} />
```

Comment:
key 에 uuid를 쓰신게 인상적이네요! 👍

Reply:
<img width="1574" alt="스크린샷 2022-01-25 오후 9 55 46" src="https://user-images.githubusercontent.com/19677717/150980777-f90f64fa-3157-4c2b-a04c-594a17074c30.png">
uuid와 비슷한 nanoid 라이브러리 예시입니다. 
렌더링할때마다 key uuid() **함수 호출**을 하게되면 재조정(reconciliation) 할때 key 비교를 통한  장점을 잃어버립니다. card  데이터 를 최초 생성할때  uuid값을 넣어주고 활용하는게 좋을듯 합니다.
```suggestion
          <Card key={card.id} card={card} />
```

Reply:
https://ko.reactjs.org/docs/reconciliation.html#recursing-on-children
<img width="1317" alt="스크린샷 2022-01-25 오후 10 04 14" src="https://user-images.githubusercontent.com/19677717/150982043-80a8731b-7762-43ee-ac76-b89c8883fba3.png">

Reply:
아~ 내용 공유해주셔서 감사합니다! @hyoungnam

Reply:
@hyoungnam 좋은 지적 감사합니다 
key의 최적화와 관련해서 착각하고 있던 부분이 있었네요
의견주신 내용 토대로 일부 변경하도록 하겠습니다